src/utils.jl

```diff
@@ -17,7 +17,7 @@ objects whose parts are the columns `codanames` in the file.
 """
 function readcoda(args...; codanames=[], kwargs...)
```

Comment:
What about replacing the type of `codanames` to a NTuple? We could then use parenthesis to list the components `readcoda("file.csv", codanames=(:A,:B,C))`.